src/main/java/org/folio/qm/service/impl/MarcRecordsServiceImpl.java

```diff
@@ -71,6 +71,7 @@ public QuickMarc findByInstanceId(UUID instanceId) {
   @Override
   public void updateById(UUID instanceId, QuickMarc quickMarc) {
     validationService.validateIdsMatch(quickMarc, instanceId);
+    validationService.validateTags(quickMarc);
```

Comment:
We have a pattern for tags in API spec. To validate it the simplest way is to add dependency 
```
<dependency>
      <groupId>org.hibernate.validator</groupId>
      <artifactId>hibernate-validator</artifactId>
    </dependency>
```
and add a related Exception handler to ConrollerAdvice class.

Reply:
Added dependency and extended ErrorHandling